bin/controller.py

```diff
@@ -4,12 +4,18 @@
 """
 
 import bottle as bt
+import re
 from pathlib import Path
 from metrics import Time
 from bin import root, config, models
 from bin.highlight import highlight, parse_language, parse_extension, languages
 
 
+BOTRE = re.compile(r'''
+    Discordbot/2\.0
+''', re.VERBOSE)
```

Comment:
I think we should have a array, in the configuration, a bit like for IPs, of "specials" UserAgents that will only receive the head (`balnk.html`).
I also think the UserAgent should be strictly verified, i.e. no regex, but a simple
```py
ua_list = ('Mozilla/5.0 (compatible; Discordbot/2.0; +https://discordapp.com)', 'Mozilla/5.0 (compatible; Mesterybot/2.0)')
if ua in ua_list:
    ...
```

for example.

Reply:
Honestly, a regexp is *just like* a `if ua in us_list`, just that it is optimized for text. With Antoine we think that the bot will be more future-proof if we use a search instead of a match.

Reply:
Like we could remove the '/2.0' to match any Discordbot

Reply:
Personally, I prefer a list of UserAgents, it's more easily configurable (via config.py) than a regex.
Also, I think the UserAgent should be strictly checked (case sensitive, etc.). In any case, if it doesn't match because of a capital letter, for example, it means it's a person trying to lie.

Reply:
> Like we could remove the '/2.0' to match any Discordbot
No, the only Discord crawler is `Mozilla/5.0 (compatible; Discordbot/2.0; +https://discordapp.com)`.

Reply:
Users can lie as much as they want with the User-Agent: if they do, they will not receive the snippet content, only the head, so it's not a problem imo

Reply:
About the configuration, we don't know if it would really add value to the product, I mean if people want we blank more bots then better we change it in the code right away, it sure will be beneficial for the entire community. If at some point we don't agree anymore to what UA should be blanked then it'll be time to make it configurable, what do you think ?

Reply:
```suggestion
    ^Mozilla/5\.0 \(compatible; Discordbot/2\.0; \+https://discord(?:app)?\.com\)$
```
in this case, can we be strict?

Reply:
> the only Discord crawler is ...
What's about other bots ? Like if people use bin on facebook/slack/... ? Are you sure we better be conservative on the UA ?

Reply:
Other bots don't use `Discordbot/2.0` in their UserAgent.